docs/publication-control.md

```diff
@@ -0,0 +1,59 @@
+# Publication control
+
+As mentioned in `OnlineBase` in our discussion about [helper models](helpers.md), anything that inherits from `OnlineBase` will have a manager that ensures that objects with `is_online == False` will be excluded from `Model.objects.all()`.
```

Comment:
Should this be single or double equals? I am not 100% sure but I feel like this is assignment rather than comparison.

Reply:
It fees like comparison to me - "if `is_online` is equal to False"